pages/index.tsx

```diff
@@ -90,7 +91,7 @@ const Home: NextPage = () => {
       );
 
       const airdropSchedules = airdropTimelines.flat().map((timeline) => ({
-        time: new Date(timeline.airdrop_window_timeline_date),
+        time: new Date(moment.utc(timeline.airdrop_window_timeline_date)),
```

Comment:
We don't need separately new Date if we are using the moment library as it provides date functionality.

Reply:
moment.utc does not return a date object and hence did this